.circleci/config.yml

```diff
@@ -362,26 +354,22 @@ workflows:
     jobs:
       - build-to-ecr:
           context: laa-assess-non-standard-magistrates-fee-dev
-      - hold-dev:
-          type: approval
-          requires:
-            - build-to-ecr
       - deploy-dev:
           context: laa-assess-non-standard-magistrates-fee-dev
-          requires:
-            - hold-dev
-      - hold-dev-crm4:
-          type: approval
           requires:
             - build-to-ecr
-      - deploy-dev-crm4:
-          context: laa-assess-non-standard-magistrates-fee-dev-crm4
-          requires:
-            - hold-dev-crm4
+          filters:
+            branches:
+              ignore:
+                - main
```

Comment:
do we also want a "main-dev" hosted env, for each repo. These "main-dev" envs, for each repo, would be the fallbacks for the dynamic branch and for other "main-dev" branches by default
i.e. 
**my-provider-branch** --> main-dev-app-store <-- main-dev-caseworker
main-dev-provider --> main-dev app store <-- **my-branch-caseworker**

merging to main would then both delete dynamic branch and automatically deploy to the main-dev host for the repo.

Reply:
That's a really good idea!